added create-eth-app, scaffold-eth, epirus, and ethereum stack exchan…

PatrickAlphaC opened the pull request:
…ge as tools to frequest. Replaced outdated cobra with brownie in the new developers start here section

And a few minor tweaks to grammar.

pi0neerpat commented:
Looks good. Some of your additions are missing the formatting " - " after the URL. Would you mind adding these and I can merge? 

Thanks for contributing!

PatrickAlphaC commented:
> Looks good. Some of your additions are missing the formatting " - " after the URL. Would you mind adding these and I can merge?
> 
> Thanks for contributing!

Rebased and updated, good call. Thank you!
